lib/Caxy/HtmlDiff/ListDiff.php

```diff
@@ -490,11 +504,12 @@ protected function diff()
      * @param string $newList
      * @param string $oldList
      * @param array $match
+     * @param array $index
      * @return string
      */
-    protected function addListElementToContent($newList, $oldList, array $match)
+    protected function addListElementToContent($newList, $oldList, array $match, array $index, $type)
```

Comment:
It looks like this function is also called on line 465 but it doesn't have these new arguments, which would cause an error.

https://github.com/caxy/php-htmldiff/blob/diff-update-no-tag-attribute-stripping/lib/Caxy/HtmlDiff/ListDiff.php#L465